src/app/system/devTools/DevMenu/DevMenu.tsx

```diff
@@ -202,6 +203,31 @@ export const DevMenu = ({ onClose = () => goBack() }: { onClose(): void }) => {
           <Spacer y={1} />
         </Flex>
 
+        <Flex mx={2}>
+          <Expandable label="Experiments" expanded={false}>
+            <Flex mx={-2} mt={1}>
+              <Text variant="xs" px={2} color="black60" italic>
+                Hint: Long press key or payload to copy them into your clipboard
+              </Text>
+              <Join separator={<Separator borderColor="black10" />}>
+                {Object.keys(experiments)
```

Comment:
To not have to `experiments[experimentKey]` 🤷 

```suggestion
                {Object.entries(experiments).reverse().map((key, experiment) => {
```